kolibri/plugins/device/assets/test/views/SelectTransferSourceModal.spec.js

```diff
@@ -6,8 +6,12 @@ function makeWrapper(options) {
   const wrapper = shallowMount(SelectTransferSourceModal, {
     ...options,
     stubs: {
-      SelectImportSourceModal: '<div data-test="select-import-source"></div>',
-      SelectDriveModal: '<div data-test="select-drive"></div>',
+      SelectImportSourceModal: {
+        template: '<div data-test="select-import-source"></div>',
```

Comment:
This is not documented in the migration guide linked in the PR description but v1 doesn't support string stubs anymore - you can check deprecation notice at the very bottom of [mount documentation](https://learningequality.slack.com/archives/CB37UM23A/p1594049471466000).